.github/workflows/additional-release-assets.yml

```diff
@@ -1,87 +0,0 @@
-
-jobs:
-  # Check if we should actually run by looking for the should-publish job
-  should-run:
-    runs-on: ubuntu-latest
-    if: ${{ github.event.workflow_run.conclusion == 'success' }}
-    outputs:
-      val: ${{ steps.check.outputs.result }}
-    steps:
-      - id: check
-        uses: actions/github-script@v6
-        with:
-          result-encoding: string
-          script: |
-            let allRuns = await github.rest.actions.listJobsForWorkflowRun({
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              run_id: context.payload.workflow_run.id,
-            });
-            for (const job of allRuns.data.jobs) {
-              if (job.name == "should-publish") {
-                if (job.status == "completed" && job.conclusion == "success") {
-                  return "run"
-                }
-              }
-            }
-            return "skip"
-
-  # Generate oranda-config-schema.json and upload it to the release
-  schema:
-    name: Add schema to release
-    env:
-      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    runs-on: ubuntu-latest
-    needs: ["should-run"]
-    if: ${{ needs.should-run.outputs.val == 'run' }}
-    steps:
-      # Setup
-      - uses: actions/checkout@v3
-        with:
-          ref: ${{ github.event.workflow_run.head_branch }}
-          fetch-depth: 0
-      # We get the tag of the release in github.event.workflow_run.head_branch
-      # No idea why it's stored there with that name, but, it is?
-      - name: Install oranda and upload schema
-        run: |
-          echo "uploading schema to ${{ github.event.workflow_run.head_branch }}"
-          curl --proto '=https' --tlsv1.2 -LsSf https://github.com/axodotdev/oranda/releases/download/${{ github.event.workflow_run.head_branch }}/oranda-installer.sh | sh
-          oranda config-schema > oranda-config-schema.json 
-          gh release upload ${{ github.event.workflow_run.head_branch }} oranda-config-schema.json
-
-  # Generate oranda.css and upload it to the release
-  oranda-css:
-    name: Add oranda-css to release
-    env:
-      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    runs-on: ubuntu-latest
-    needs: ["should-run"]
-    if: ${{ needs.should-run.outputs.val == 'run' }}
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          ref: ${{ github.event.workflow_run.head_branch }}
-          fetch-depth: 0
-      - name: Install oranda and build CSS
-        run: |
-          echo "uploading CSS to ${{ github.event.workflow_run.head_branch }}"
-          curl --proto '=https' --tlsv1.2 -LsSf https://github.com/axodotdev/oranda/releases/download/${{ github.event.workflow_run.head_branch }}/oranda-installer.sh | sh
-          oranda generate-css
-          ls oranda-css/dist
-          echo "built css!"
-          # Upload css to the Github Release™
-          gh release upload ${{ github.event.workflow_run.head_branch }} oranda-css/dist/oranda.css
-          echo "uploaded css!"

```

Comment:
it doesn't look like you added generation for oranda-css, and this is the Really Important one to migrate.

Reply:
Added.